app/src/main/java/com/arkbuilders/arkdrop/presentation/feature/transferconfirmation/TransferConfirmation.kt

```diff
@@ -0,0 +1,230 @@
+package com.arkbuilders.arkdrop.presentation.feature.transferconfirmation
```

Comment:
Currently the content of this screen is not fully displayed in my phone (Pixel 3a), as in this screenshot:

<img width="398" alt="Screenshot 2024-05-01 at 18 31 58" src="https://github.com/ARK-Builders/ARK-Drop/assets/4281969/122eff0d-8410-4e19-b3f3-e9b8ea26ad7f">

The `Hash Code` text is hidden.
Can we make the layout scrollable or something, to make sure all the contents are visible to user?

Reply:
I've updated and made it scrollable. Thank you